packages/dui3/store/hostApp.ts

```diff
@@ -446,28 +446,89 @@ export const useHostAppStore = defineStore('hostAppStore', () => {
   const refreshSendFilters = async () =>
     (sendFilters.value = await app.$sendBinding?.getSendFilters())
 
-  const getSendSettings = async () =>
-    (sendSettings.value = await app.$sendBinding.getSendSettings())
+  const getSendSettings = async () => {
+    sendSettings.value = await app.$sendBinding.getSendSettings()
+    tryToUpgradeSettings('SenderModelCard')
```

Comment:
This is a bit of a smell: 
- either rename to `getSendSettingsAndUpgradeCards()`, or
- leave as is, and call `tryToUpgradeSettings` at the end in `initializeApp` (this would be my preference)

Reply:
that sounds make sense, will do second